.github/workflows/onPRReviews.yml

```diff
@@ -1,23 +1,120 @@
 name: On review update
 on:
   pull_request:
-    types: [review_requested, review_request_removed]
+    types: [ready_for_review, review_requested, review_request_removed]
   pull_request_review:
     types: [edited, dismissed, submitted]
 jobs:
+  getPRNumber:
```

Comment:
camel case here but kebab case on the other PR 

I don't mind which, but we should make them all consistent